cmd/melt/meltini.go

```diff
@@ -0,0 +1,61 @@
+package melt
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var meltMelitiniCmd = &cobra.Command{
+	Use:              "meltini",
+	Short:            "uses meltini APIs to generate metrics from a template",
+	Long:             `Uses meltini APIs to generate OT data from provided stated template file.`,
+	TraverseChildren: true,
+	Run:              meltMeltini,
+}
+
+func init() {
+	meltMelitiniCmd.Flags().String("template-file", "", "path to the stated template file")
+	meltMelitiniCmd.Flags().String("meltini-url", "http://localhost:3000/", "meltini REST APIs URL")
```

Comment:
Should we think about maybe importing MELTini as a dependency (I will have to make it available on npm)? Hitting that endpoint means we'll be starting a node server on local.